languages/elixir/exercises/concept/chessboard/.docs/instructions.md

```diff
@@ -0,0 +1,31 @@
+As a chess enthusiast, you would like to write your own version of the game. Yes, there maybe plenty of implementations of chess available online already, but yours will be unique!
+
+But before you can let your imagination run wild, you need to take care of the basics. Let's start by generating the board.
+
+Each square of the chessboard is identified by a letter-number pair. The vertical columns of squares, called files, are labeled A through H. The horizontal rows of squares, called ranks, are numbered 1 to 8.
+
+## 1. Define the rank range
+
+Implement the `rank_range/0` function. It should return a range of integers, from 1 to 8.
+
```

Comment:
Putting the expected return value here would spoil this step...

Reply:
I suggest still adding the method called, just without the return.

```suggestion

```elixir
Chessboard.ranks()
```
```